packages/lab/src/components/Flow/DroppableFlow.tsx

```diff
@@ -278,6 +278,9 @@ export const HvDroppableFlow = ({
           defaultEdgeOptions={defaultEdgeOptions}
           snapGrid={[1, 1]}
           snapToGrid
+          onError={(code, message) => {
+            console.error(message);
```

Comment:
We should wrap this in an `if (import.meta.env.DEV) {}` block, and eslint-ignore it

Reply:
👍